pysatNASA/instruments/cnofs_vefi.py

```diff
@@ -109,23 +96,44 @@ def init(self):
     return
 
 
-def clean(inst):
+def clean(self):
     """Routine to return VEFI data cleaned to the specified level
 
-    Parameters
-    -----------
-    inst : pysat.Instrument
-        Instrument class object, whose attribute clean_level is used to return
-        the desired level of data selectivity.
-
     Note
     ----
     'dusty' or 'clean' removes data when interpolation flag is set to 1
```

Comment:
Do we want to replace the data with nans instead of removing as in other cnofs cleaning routines?

Reply:
I'm not familiar enough with vefi, but looking at cdaweb it looks like the interpolation flag is part of the B field data. And looking at the cnofs_vefi instrument, it looks like it only supports the bfield data anyway. So I think it's fine to remove all of the data.

Reply:
For the other cnofs instruments, we don't have multiple data products that require different cleaning parameters.